msg/sensor_header.msg

```diff
@@ -0,0 +1,4 @@
+uint32 version
+uint64 reading_time_utc_s
+uint64 reading_uptime_millis
+uint64 sensor_reading_time_s
```

Comment:
what is the difference between the first and third one?

Reply:
or I suppose more, what is the intended use of these two?

Reply:
First is from the mote/Bristlemouth
Third is if the sensor (i.e. Aanderaa) is able to generate timestamps